.github/workflows/ci.yaml

```diff
@@ -356,7 +357,10 @@ jobs:
       - name: Install CodeNotary
         shell: bash
         run: |
-          bash <(curl https://getvcn.codenotary.com -L)
+          mkdir -p "${{ github.workspace }}/bin"
```

Comment:
`github.workspace` seems to have worked :+1: 
But we now have the `vcn` action helper for this that should be used instead :see_no_evil: